Code/Editor/EditorEngineProcessFramework/EngineProcess/Implementation/WorldRttiConverterContext.cpp

```diff
@@ -199,7 +199,10 @@ void ezWorldRttiConverterContext::UnregisterObject(const ezUuid& guid)
 
 ezRttiConverterObject ezWorldRttiConverterContext::GetObjectByGUID(const ezUuid& guid) const
 {
-  ezRttiConverterObject object = ezRttiConverterContext::GetObjectByGUID(guid);
+  ezRttiConverterObject object = ezRttiConverterContext::GetObjectByGUID(guid);
+
+  if (!guid.IsValid())
+    return object;
```

Comment:
Not sure why this was necessary. Apparently we get invalid GUIDs here and this could now crash a bit below.

Reply:
@Sanakan8472 any idea?

Reply:
This shouldn't happen unless the serialization was messed up. Do you have a callstack / repro case for this?

Reply:
Ah nevermind, it's all the derived type changes. Looks fine.